addons/web/static/tests/core/commands/command_palette_tests.js

```diff
@@ -1547,3 +1547,49 @@ QUnit.test("generate new session id when opened", async (assert) => {
     await nextTick();
     assert.equal(lastSessionId, 1);
 });
+
+QUnit.test("checks that href is correctly used", async (assert) => {
+    await mount(TestComponent, target, { env });
+    const providers = [
+        {
+            namespace: "@",
+            provide: () => [
+                {
+                    name: "Command with link",
+                    action: () => {
+                        assert.step("a");
+                    },
+                    href:"https://www.odoo.com",
+                },
+                {
+                    name: "Command2",
+                    action: () => {},
+                },
+            ],
+        },
+    ];
+    const config = {
+        providers,
+    };
+    env.services.dialog.add(CommandPalette, {
+        config,
+    });
+    patchWithCleanup(window, {
+        open: (href) => {
+            assert.step(href.toString());
+        }
+    })
+    await nextTick();
+    await editSearchBar("@");
+    const command = target.querySelector(".o_command_palette .o_command");
+    assert.strictEqual(command.firstChild.getAttribute("href"), "https://www.odoo.com");
```

Comment:
"firstChild" seems a bit weak, as it will break if we change the command DOM structure.
Shouldn't we target the first a tag within the command instead?